src/plugins/memory/memory-editor.js

```diff
@@ -1,11 +1,98 @@
-import React from 'react';
+import { SIZE } from './constants.js';
+import React, { useState } from 'react';
+import { Button, Form, Radio } from 'antd';
+import { useTranslation } from 'react-i18next';
+import cloneDeep from '../../utils/clone-deep.js';
+import { resizeTilePairs } from './memory-utils.js';
+import { FORM_ITEM_LAYOUT } from '../../domain/constants.js';
+import MarkdownInput from '../../components/markdown-input.js';
 import { sectionEditorProps } from '../../ui/default-prop-types.js';
 
-// eslint-disable-next-line no-unused-vars
+const FormItem = Form.Item;
+const RadioGroup = Radio.Group;
+const RadioButton = Radio.Button;
+
 function MemoryEditor({ content, onContentChanged }) {
+  const { t } = useTranslation('memory');
+
+  const { title, size, tilePairs } = content;
```

Comment:
Actually I'm considering dropping the `title`, given the `markdown` plugin can always be used for title and/or description. I've also made a list of where we stand with the common plugin fields and we're a bit mixed with `title` though most plugins actually don't have it:
https://educandu.atlassian.net/wiki/spaces/WIKI/pages/145096705/Plugins+data+models

Reply:
I agree, `width` would be more consistent and also bring functionality not available otherwise. If editors want a header, there's the `markdown` plugin for it.

Reply:
Ok then, I've dropped `title` and added `width`.

Reply:
:+1: